change/@ni-nimble-components-48e9cade-64e1-47a5-a616-c2e2a930a2b1.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Align column header based on cell content alignment",
```

Comment:
Could you ensure we have a storybook test that shows both the sorting and grouping header icons in both left and right aligned columns? I'd be fine if that's a new test or a change to the existing numeric column matrix test.